docs/source/ecf/guidance.html.md

```diff
@@ -198,19 +215,32 @@ For more detailed information see the specifications for this [find a delivery p
  POST /api/v3/partnerships/ecf
 ```
 
-An example request body is listed below. Request bodies must include all necessary data attributes, namely `cohort`, `school_id` and `delivery_partner_id`.
+Request bodies must include the following data attributes:  
+ 
+* `cohort` 
+* `school_id` 
+* `delivery_partner_id` 
 
-An example response body is listed below. Successful requests will return a response body with updates included.
+Successful requests will return a response body with updates included. 
 
-#### Providers should note:
+#### Do providers need to take any action to continue existing partnerships with schools from one cohort to the next? 
 
-* the API assumes schools intend to work with a given provider for consecutive cohorts. School induction tutors will be prompted to confirm existing partnerships with providers **will continue** into the upcoming cohort. Providers do not need take any action to continue existing partnerships with schools from one cohort to the next
-* in order for new providers to confirm partnerships with schools for an upcoming cohorts, school induction tutors must first notify DfE that their schools **will not continue** their former partnerships with existing providers for the upcoming cohort. Until induction tutors have done this, any new partnerships with new providers will be rejected by the API
-* you can use POST to create a new partnership once one has been rejected
+No, the API assumes schools intend to work with a given provider for consecutive cohorts. School induction tutors will be prompted to confirm existing partnerships with providers will continue into the upcoming cohort. 
 
-For more detailed information see the specifications for this [confirm an ECF partnership endpoint.](/api-reference/reference-v3.html#api-v3-partnerships-ecf-post)
+#### What happens if schools have previous partnerships with other lead providers? 
 
-#### Example request body:
+For new providers to confirm partnerships with schools for an upcoming cohort, school induction tutors must first notify DfE that their schools will not continue their former partnerships with existing providers for the upcoming cohort. Until induction tutors have done this, any new partnerships with new providers will be rejected by the API. 
+
+#### What if a school selected 'school-led' by mistake?  
+
+If a school has mistakenly selected a `school-led` induction programme and intended to choose `provider-led`, lead providers can correct this using the `POST partnerships` endpoint. Providers must:  
```

Comment:
hmm I'm not sure about this, I know we reject partnering with those kind of schools?